app/models/linkage_group.rb

```diff
@@ -0,0 +1,14 @@
+class LinkageGroup < ActiveRecord::Base
+
+  has_and_belongs_to_many :linkage_maps,
+                          join_table: 'map_linkage_group_lists',
+                          foreign_key: 'linkage_group_id',
+                          association_foreign_key: 'linkage_map_id'
```

Comment:
Don't want to cram too many changes into one PR - for now I'm setting up model classes; you are right in that some FK names should be fixed but it's not as simple as renaming columns (I'll need to generate additional numerical ID columns and set them up as keys since the existing keys are strings). I intend to do this as a separate task. You are correct about the association FK designation; that's been fixed.